src/Codeception/Lib/Console/Output.php

```diff
@@ -76,7 +76,7 @@ protected function clean(string $message): string
         return $message;
     }
 
-    public function debug($message): void
+    public function debug(string $message): void
```

Comment:
`debug` can print any type of data (see [Debug class](https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Util/Debug.php#L28))